.gitignore

```diff
@@ -9,7 +9,6 @@ __pycache__
 *.srl
 *.req
 *.log
-*.jar
```

Comment:
We absolutely can not support JARs checked into source control. Let's discuss this before propagating this change any further.

Reply:
@aayars : ok, please back it out!  We'll work with @apurvam on a proper solution